### DISCUSSION TITLE
How to use environment variable?

### DISCUSSION THREAD

**Discussion by jiangzm** [postmanlabs/postman-flows#103] [Q&A]
how to get an environment variables assign to local variables in the postman flows?

**Comment by samay-v** [comment]
Hey @jiangzm, Could you give an example of your usecase so that I can understand it better?

**Comment by hahnavi** [comment]
Hi @samay-v ,
I want to use an environment variable in the Validate block and I tried using double curly braces but it doesn't work.
Can we use environment variables on flows blocks?

![2021-11-24_20-53](https://user-images.githubusercontent.com/743718/143251376-879b926a-9e94-424c-9fad-7571ab894105.png)
